spec/services/doc_auth/socure/requests/document_request_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe DocAuth::Socure::Requests::DocumentRequest do
+  let(:document_capture_session_uuid) { 'abc123' }
+  let(:redirect_url) { 'https://somewhere.com' }
+  let(:language) { :en }
+
+  subject(:document_request) do
+    described_class.new(document_capture_session_uuid:, redirect_url:, language:)
+  end
+
+  describe 'a new request' do
+    it 'exists' do
+      expect(document_request).to be
+    end
+  end
+
+  describe '#fetch' do
+    let(:language) { :en }
+    let(:document_type) { 'license' }
+    let(:fake_socure_endpoint) { 'https://fake-socure.com/' }
+    let(:socure_document_capture_url) { 'https://verify.socure.us/something' }
+    let(:docv_transaction_token) { 'docv transaction token' }
+    let(:fake_socure_response) do
+      {
+        referenceId: 'socure-reference-id',
+        data: {
+          eventId: 'socure-event-id',
+          customerUserId: '121212',
+          docvTransactionToken: docv_transaction_token,
+          qrCode: 'data:image/png;base64,iVBO......K5CYII=',
+          url: socure_document_capture_url,
```

Comment:
this is the value for our redirect url ... should we use an fake domain instead to avoid any false expections by someone who may follow behind:
```suggestion
          url: 'https://capture-my-images-app.example.com',
```

Reply:
I don't understand what fake expectations you mean. We expect to get our redirect url here in the actual Socure response, yes?